migrations/versions/d25d457860de_lowercase_discount_code.py

```diff
@@ -0,0 +1,34 @@
+"""lowercase discount code
+
+Revision ID: d25d457860de
+Revises: 6f7b6fad3f54
+Create Date: 2019-05-19 02:06:14.730180
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = 'd25d457860de'
+down_revision = '6f7b6fad3f54'
+
+
+def upgrade():
+    op.execute("UPDATE discount_codes SET code = concat(code, '_') where code not in (SELECT DISTINCT ON (upper(code)) code FROM discount_codes);",
```

Comment:
We want only per event discount codes to be case sensitive unique.

I should not be able to create `abC` and `AbC` for the same event, but should be able to do it for another event. This will update all discount codes globally. Please follow the discussion  on previous PR

Reply:
@iamareebjamal Got it. Will make the required changes.

Reply:
@iamareebjamal So, in this case, I don't think citext would the correct thing to go with. We can first simply add a `_` where the discount code and event_id both are same. But how can we apply the unique constraint for columns with same event_id. Citext would apply the constraint globally.

Reply:
CITEXT does not apply unique constraint. If you add unique constraint on event_id and discount_code, everything will work correctly. Please proceed with fixing the update query and also add unique constraint to (event_id, discount_code)

And no need to write create extension. It's already there in previous migraions